modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/helper/TranslationRequestHelper.java

```diff
@@ -199,10 +200,16 @@ public long[] getModelClassPKs() throws PortalException {
 		long[] modelClassPKs = new long[keys.length];
 
 		for (int i = 0; i < keys.length; i++) {
+			GroupKeyInfoItemIdentifier groupKeyInfoItemIdentifier =
+				new GroupKeyInfoItemIdentifier(getGroupId(), keys[i]);
+
+			groupKeyInfoItemIdentifier.setVersion(
```

Comment:
Yes @AliciaGarciaGarcia, that's a problem with Web Content. 

For example when you are in the interface and see an expired or scheduled web content, if we take the latest approved, then we wouldn't be exporting the translations for the ones we are displaying

Reply:
But putting it on the `TranslationRequestHelper` you are adding it for all the objects that can be translated instead of letting them to decide the version 🤔